checker/src/test/java/org/checkerframework/checker/test/junit/ainferrunners/README.md

```
@@ -1,4 +1,4 @@
-This package contains the test runners for testing the -Ainfer command-line
+This package contains the test runners for testing the `-Ainfer` command-line
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)